cypress/integration/spec.js

```diff
@@ -1,15 +1,25 @@
 const fetchMock = require('fetch-mock')
 
-describe('Chart Explorer', function() {
-  it('Visits the app', function() {
-    const visitOptions = {
+describe('Chart Builder', function() {
+  let visitOptions
+  beforeEach(function() {
+    visitOptions = {
       onBeforeLoad: win => {
         let fetch = fetchMock.sandbox()
         fetch.get('https://api.data.world/v0/user', this.profile)
         fetch.post(
           'https://api.data.world/v0/sql/data-society/iris-species?includeTableSchema=true',
           this.queryResponse
         )
+        fetch.post(
```

Comment:
what is the purpose of this addition?

Reply:
I added a test to share a chart as an insight. That test calls the `https://api.data.world/v0/insights/data-society/iris-species` endpoint so I had to mock that over here 👍